hacs.json

```diff
@@ -1,4 +1,5 @@
 {
-    "name": "apexcharts-card",
-    "render_readme": true
+  "homeassistant": "2023.4.0b0",
```

Comment:
Ensure backward compatibility for older versions (or: only show this version of Apexcharts to users with HA2023.4.0b0 or higher)